### DISCUSSION TITLE
Interactive Rebase

### DISCUSSION THREAD

**Discussion by bgoewert** [Murmele/Gittyup#418] [Ideas]
It does not look like that is possible yet, correct? Are there plans to add the ability to `git rebase --interactive`? As an example, Fork (git-fork) has an excellent means of doing this that I used all the time when I previously used a system they supported (win/mac). If need be I can get a screenshot of that functionality.

**Comment by Murmele** [comment]
Hi @bgoewert,

currently we have only solving of conflict during rebase, but not interactive directly.

**Comment by bgoewert** [comment]
Thanks for replying to this @Murmele. I created a ticket to track this #420. Like I said in the ticket, it's easy enough to open a terminal and perform this. This feature would be a nice-to-have. I don't know if Gittyup is meant to be a full featured git client. But, for me, this is the only thing that is missing.

**Comment by Murmele** [comment]
@bgoewert so you might also just open the terminal from the gui and doing git rebase --interactive?

**Reply by bgoewert** [reply]
Correct. And that is what I am currently doing. Like I said, it is a nice-to-have feature. However, with this being a graphical git client, I would have expected Gittyup to have support for this. Otherwise, why use a graphical git client at all.

**Reply by Murmele** [reply]
Would you use the graphical git client only for interactive rebase? I never did an interactive rebase. There are a lot of usecases where the graphical interface has benefits over the terminal, for example staging single lines of a file, ...

**Reply by bgoewert** [reply]
If Gittyup supported interactive rebase, I would use Gittyup over the terminal. Given the inter-activeness, I would assume that 
interactive rebase is more suited to a graphical interface. Maybe someone else can weigh in on this too and give their input.

**Reply by thetruegamer** [reply]
Hopping in on the interactive rebase train: I'd use it too. I previously used GitExtensions on Windows, which also has the interactive rebase in GUI mode, if you want to check it out too :)

**Comment by bgoewert** [comment]
Just as a reference, here is how GitKraken handles interactive rebase in their GUI.
https://www.gitkraken.com/learn/git/problems/git-interactive-rebase

**Comment by LinuxDevon** [comment]
I want to tag in this would be useful for me as well. I use the Fork GUI client on Mac and like the way they do it: https://git-fork.com/?ref=localhost
